Formula/dsda-doom.rb

```diff
@@ -0,0 +1,58 @@
+class DsdaDoom < Formula
+  desc "Fork of prboom+ with a focus on speedrunning"
+  homepage "https://github.com/kraflab/dsda-doom"
+  url "https://github.com/kraflab/dsda-doom/archive/refs/tags/v0.24.3.tar.gz"
+  sha256 "d4cfc82eea029068329d6b6a2dcbe0b316b31a60af12e6dc5ad3e1d2c359d913"
+  license "GPL-2.0-only"
+
+  head do
+    url "https://github.com/kraflab/dsda-doom.git", branch: "master"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "fluid-synth"
+  depends_on "libvorbis"
+  depends_on "mad"
+  depends_on "portmidi"
+  depends_on "sdl2"
+  depends_on "sdl2_image"
+  depends_on "sdl2_mixer"
+
+  on_linux do
+    depends_on "pcre"
```

Comment:
Can we use `pcre2`?

You can also move this outside the `on_linux` block, since there is `pcre` linkage on macOS as well.

Reply:
There doesn't seem to be `pcre2` compatibility unfortunately.